updating/index.adoc

```diff
@@ -0,0 +1,91 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understanding-container-platform-updates']
+== Understanding OpenShift Container Platform updates
+xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[About the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs.
```

Comment:
The same issue with https://github.com/openshift/openshift-docs/pull/43645#discussion_r840192502

Reply:
Addressed

Reply:
Addressed